README.md

```diff
@@ -65,6 +65,7 @@ npm install wakatime-client --save
 ```javascript
 import { WakaTimeClient } from 'wakatime-client';
 
+// Optionally, you can specify a custom API URL as a second parameter
```

Comment:
Thanks for updating the documentation, do you mind adding a fake code example as well like

```javascript
const client = new WakaTimeClient('some api key', 'https://hakatime-demo.mtx-dev.xyz/api/v1/');
```

Reply:
Sure! However, as I'm the maintainer of Wakapi, I'm probably going to put a Wakapi URL instead 😉